### PR TITLE
fixed logging basicConfig issue and tests for Windows

### LINKED ISSUE #128: Logging uses basicConfig which will overwrite config of a package which imports pycondor
#### Description
If a package <my_package> imports and uses pycondor, the use of basicConfig in pycondor.utils will overwrite calls to basicConfig in <my_package>

#### Expected results
Logging like:
` '%(levelname)s: my_package - %(name)s : %(message)s' `

#### Actual results
Logging like:
` '%(levelname)s: pycondor - %(name)s : %(message)s' `

### PR DESCRIPTION
**Pull request by khornlund**
#### Reference Issue

Fixes #128 

#### What does this pull request implement/fix? Explain your changes.

Changes utils:_setup_logger to specify a stdout StreamHandler and no longer uses basicConfig.

Also fixed a path separator issue in test_dagman

### CODE REVIEW

File: pycondor/utils.py [hunk 1]
```diff
@@ -50,6 +49,9 @@ def _setup_logger(cls, verbose=0):
                        'Valid options are {}.'.format(
                            verbose, cls.name, logging_level_dict.keys()))
     logger = logging.getLogger(cls.name)
+    sh = logging.StreamHandler(sys.stdout)
```
Comment (jrbourbeau) [inline pycondor/utils.py]: Nitpicky comment: can you give this a more descriptive name? Perhaps something like `stream_handler` instead of `sh`

Comment (khornlund) [reply pycondor/utils.py]: No problem.

File: pycondor/tests/test_dagman.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def test_job_arg_name_files(tmpdir, fancyname):
 
     # Get root of the dagman submit file (submit file basename w/o .submit)
     submit_file_line = dagman_submit_lines[0]
-    submit_file_basename = submit_file_line.split('/')[-1].rstrip()
+    submit_file_basename = submit_file_line.split(os.path.sep)[-1].rstrip()
```
[the remark this answers is not in the document]
Comment (khornlund) [reply pycondor/tests/test_dagman.py]: Whoops - how weird.

**Comment by jrbourbeau** [comment]
Thanks as always @khornlund!

**Pull request merged**